File: Hotel/csharp/HOTELINTERFAZ/HOTELINTERFAZ/HOTELINTERFAZ/Models/Reserva.cs
using System;
using System.Text.Json.Serialization;

namespace HOTELINTERFAZ.Models
{
    public class Reserva
    {
        [JsonPropertyName("_id")]
        public string Id { get; set; }

        [JsonPropertyName("clienteId")]
        public string ClienteId { get; set; }

        [JsonPropertyName("habitacionId")]
        public string HabitacionId { get; set; }

        [JsonPropertyName("fechaEntrada")]
        public DateTime FechaEntrada { get; set; }

        [JsonPropertyName("fechaSalida")]
        public DateTime FechaSalida { get; set; }

        [JsonPropertyName("personas")]
        public int Personas { get; set; }

        [JsonPropertyName("precioTotal")]
        public double PrecioTotal { get; set; }

        [JsonPropertyName("cancelacion")]
        public bool Cancelacion { get; set; }
    }
}

File: Hotel/kotlin/app/src/main/java/com/example/aplicacion_hotel/Model/Reserva.kt
package com.example.aplicacion_hotel.Model

data class Reserva(
    val _id: String,
    val habitacionId: String,
    val fechaEntrada: String,
    val fechaSalida: String,
    val personas: Int,
    val precioTotal: Double,
    val cancelacion: Boolean
)